### PR TITLE
[SERVICES][SETUPLIB][UMANDLG][USETUP] Add missing \n to DPRINT() calls

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Improve debug log.

## Proposed changes

- Add missing \n to DPRINT() calls
- Add promote 1 DPRINT().

### CODE REVIEW

File: base/setup/usetup/usetup.c [hunk 1]
```diff
@@ -3060,7 +3060,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
         Status = InitDestinationPaths(&USetupData, InstallDir, InstallPartition);
         if (!NT_SUCCESS(Status))
         {
-            DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx", Status);
+            DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx\n", Status);
```
Comment (JoachimHenze) [inline base/setup/usetup/usetup.c:3063]:
```suggestion
            DPRINT1("InitDestinationPaths() failed. Status 0x%lx\n", Status);
```
for shortness and consistency to the line above in partlist.c

File: base/setup/usetup/usetup.c [hunk 2]
```diff
@@ -3164,7 +3164,7 @@ InstallDirectoryPage(PINPUT_RECORD Ir)
             Status = InitDestinationPaths(&USetupData, InstallDir, InstallPartition);
             if (!NT_SUCCESS(Status))
             {
-                DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx", Status);
+                DPRINT1("InitDestinationPaths() failed. Status code: 0x%lx\n", Status);
```
Comment (JoachimHenze) [inline base/setup/usetup/usetup.c:3167]:
```suggestion
                DPRINT1("InitDestinationPaths() failed. Status 0x%lx\n", Status);
```

File: base/applications/utilman/umandlg/process.c [hunk 3]
```diff
@@ -84,7 +84,7 @@ BOOL IsProcessRunning(IN LPCWSTR lpszProcessName)
     hProcess = OpenProcess(SYNCHRONIZE, FALSE, dwProcessID);
     if (!hProcess)
     {
-        DPRINT("IsProcessRunning(): Failed to open the process! (Error: %lu)", GetLastError());
+        DPRINT("IsProcessRunning(): Failed to open the process! (Error: %lu)\n", GetLastError());
```
Comment (JoachimHenze) [inline base/applications/utilman/umandlg/process.c:87]:
```suggestion
        DPRINT("Failed to open the process! (Error: %lu)\n", GetLastError());
```
DPRINT logs the function name itself already.

File: base/applications/utilman/umandlg/process.c [hunk 4]
```diff
@@ -211,7 +211,7 @@ BOOL CloseProcess(IN LPCWSTR lpszProcessName)
     hProcess = OpenProcess(PROCESS_TERMINATE, FALSE, dwProcessID);
     if (!hProcess)
     {
-        DPRINT("CloseProcess(): Failed to open the process for termination! (Error: %lu)", GetLastError());
+        DPRINT("CloseProcess(): Failed to open the process for termination! (Error: %lu)\n", GetLastError());
```
Comment (JoachimHenze) [inline base/applications/utilman/umandlg/process.c:214]:
```suggestion
        DPRINT("Failed to open the process for termination! (Error: %lu)\n", GetLastError());
```
DPRINT logs the function name itself already.

File: base/setup/usetup/usetup.c (hunk 1 above)
Comment (SergeGautherie) [reply base/setup/usetup/usetup.c:3063]: Removed.

File: base/applications/utilman/umandlg/process.c (hunk 3 above)
Comment (HBelusca) [reply base/applications/utilman/umandlg/process.c:87]: No, it just logs the source file path and line number.

File: base/applications/utilman/umandlg/process.c (hunk 4 above)
Comment (HBelusca) [reply base/applications/utilman/umandlg/process.c:214]: no it doesn't.

File: base/setup/usetup/usetup.c (hunk 1 above)
Comment (HBelusca) [reply base/setup/usetup/usetup.c:3063]: ok

File: base/setup/usetup/usetup.c (hunk 2 above)
Comment (HBelusca) [reply base/setup/usetup/usetup.c:3167]: ok

File: base/applications/utilman/umandlg/process.c (hunk 3 above)
Comment (SergeGautherie) [reply base/applications/utilman/umandlg/process.c:87]: Really?

Comment (JoachimHenze) [reply base/applications/utilman/umandlg/process.c:87]: You are right. It doesn't. My fault. Resolving this request.

File: base/applications/utilman/umandlg/process.c (hunk 4 above)
Comment (JoachimHenze) [reply base/applications/utilman/umandlg/process.c:214]: Indeed, it doesn't. My fault. Resolved.

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**